src/main/java/frc/robot/subsystems/arm/ArmStageTwo.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.*;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.ArmConstants;
+
+public class ArmStageTwo extends SubsystemBase {
+  private CANSparkMax armMotorPrimary;
+  private CANSparkMax armMotorSecondary;
+  private RelativeEncoder encoder;
```

Comment:
is this how you instantiate a through bore encoder plugged into a sparkmax? does it need to be instantiated? are we configuring the sparkmax hardware correctly for a throughbore encoder in the dataport?(idk if this is done via the client or API, but not doing it fries the sparkmax)